biom/table.py

```diff
@@ -3713,6 +3808,105 @@ def any_value(vals, id_, md):
 
         return t
 
+    def to_dataframe(self):
+        """Convert matrix data to a Pandas SparseDataFrame
+
+        Notes
```

Comment:
hmm.... okay, thanks, good catch. ill issue a quick PR on this to fix locations here and in the other places you noted. Thanks!!